cypress/integration/free_trial_page.js

```diff
@@ -34,6 +34,7 @@ describe('The free trial page', () => {
     cy.visit('/free-trial/');
 
     cy.get('#get-instance-email-input').type('test@gmail.com');
+    cy.get('#scm').select('GitLab Cloud');
```

Comment:
Fwiw I don't think this is necessary, the warning is displayed regardless of scm.

Reply:
I was thinking that by simulating the usage of the form.. it might work.. for e.g. maybe a debounce on the email input field means the error has not appeared before the assertion is made. I was hoping that selecting the next field might cause the validation to be done before the assertion.. but its a wild guess.